src/py/mains/run.py

```diff
@@ -87,6 +89,9 @@ def wait_for_file(filename, task, job_name = ""):
             if support.job_type == "SGE":
                 fc_run_logger.info( "deleting the job by `qdel` now..." )
                 os.system("qdel %s" % job_name) # Failure is ok.
+            if support.job_type == "SLURM"
+            	fc_run_logger.info( "Deleting the job by 'scancel' now...")
```

Comment:
> so I used the terrible in line text editor Git provides via the HTML website.

Interesting. GitHub usually gets this stuff right, and Ace is a pretty decent editor. Maybe it didn't detect the indentation because of mistakes elsewhere in the code. I'll strip them all soon.

* https://github.com/blog/905-edit-like-an-ace

For code which already has tabs, you can alter Ace rendering using the Stylish extension to Firefox with this Userstyle:
* https://userstyles.org/styles/70979/github-better-sized-tabs-in-code